src/Opulence/Databases/Migrations/Migrator.php

```diff
@@ -52,7 +53,13 @@ public function rollBackAllMigrations() : array
         // These classes are returned in chronologically descending order
         $migrationClasses = $this->executedMigrations->getAll();
         $migrations = $this->resolveManyMigrations($migrationClasses);
-        $this->executeRollBacks($migrations);
+        try {
+            $this->executeRollBacks($migrations);
+        } catch (Exception $e) {
+            $this->connection->rollBack();
+
+            return [];
```

Comment:
Should we instead let the exception bubble up?  Otherwise, you're not going to have any insights into what actually broke.